deployment/docker-compose/template.backend.env

```diff
@@ -5,3 +5,7 @@ POSTGRES_PORT=5432
 POSTGRES_DB=postgres
 
 PROMETHEUS_MULTIPROC_DIR="/tmp"
+
+LLM_MODEL="ollama/llama3:70b" # "gpt-4o-mini" for OpenAI
```

Comment:
I was thinking that the prod deployment would be 70b so left that in the template. When I made a copy for myself, I made it 3.2:1b instead for running locally. What do you think? Is it confusing?

Reply:
My gut says that minimizing hurdles in dev is a higher prio for now. 

It's a small difference either way, but this just adds one more small stumbling block/ reason for a call to fail if we're trying to do dev work - took me a few mins to debug (especially because when I first glanced at logs it said "llama3" and I thought "Oh I've already got that running!")

Reply:
Makes sense! I'll fix it